tests/acceptance/customCommands/findItemInFilesList.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Find an item in the files list. Scrolls down in case the item is not visible in viewport
+ */
+exports.command = async function (itemName) {
+  // Escape double quotes inside of selector
+  if (itemName.indexOf('"') > -1) {
+    itemName = itemName.replace(/\\([\s\S])|(")/g, '\\$1$2')
+  }
+
+  await this.executeAsync(function (itemName, done) {
+    const filesListContainer = document.querySelector('#files-list-container')
```

Comment:
generic would be nice, but looks like it will be more complex to add all these selectors. so I guess keep it for now